src/Illuminate/Foundation/Console/ModelMakeCommand.php

```diff
@@ -210,6 +237,7 @@ protected function getOptions()
             ['resource', 'r', InputOption::VALUE_NONE, 'Indicates if the generated controller should be a resource controller'],
             ['api', null, InputOption::VALUE_NONE, 'Indicates if the generated controller should be an API controller'],
             ['requests', 'R', InputOption::VALUE_NONE, 'Create new form request classes and use them in the resource controller'],
+            ['table', 't', InputOption::VALUE_REQUIRED, 'Sets the table name for the model.'],
```

Comment:
Does that mean it's required? Why?

Reply:
It means that if you use the table option, the value for the option is required i.e. the name of the table.